File: 07_covid home/200317_Collection/example#1/java_script/after.js
function getRandomInt(min, max) {
		
	min = Math.ceil(min);
	max = Math.floor(max);
	return Math.floor(Math.random() * (max - min + 1)) + min;
}


function getFruit(){

	const fruits = ['Mango', 'Apple', 'Orange', 'Banana']

	const i = getRandomInt(0, fruits.length)

	return fruits[i]

}


console.log(getFruit())

File: 07_covid home/200317_Collection/example#1/java/after.java
import java.util.Random;

public class Main
{
    
	public static String getFruit() {
        
		String fruits[] = {"Mango", "Apple", "Orange", "Banana"};
        
		int random = (new Random()).nextInt() & Integer.MAX_VALUE;
        
		int i = random % fruits.length;
        
		return fruits[i];        
	}
    
	public static void main(String[] args) {
	    
	        System.out.println(getFruit());
	}
}
